docs/storage.md

```diff
@@ -150,3 +150,50 @@ promtool tsdb create-blocks-from openmetrics <input file> [<output directory>]
 ```
 
 After the creation of the blocks, move it to the data directory of Prometheus. If there is an overlap with the existing blocks in Prometheus, the flag `--storage.tsdb.allow-overlapping-blocks` needs to be set. Note that any backfilled data is subject to the retention configured for your Prometheus server (by time or size).
+
+## Backfilling for Recording Rules
+
+### Overview
+
+When a new recording rule is created there is no historical data for it, rather recording rule data only exists from the creation time on. However there is a `promtool` subcommand that makes it possible to create historical recording rule data.
+
+### Usage
+
+```
```

Comment:
This copy and paste seems fragile, I would remove it.

Reply:
Maybe say: To see all the options, use `promtool tsdb create-blocks-from rules --help`

Reply:
We could setup auto-generated one, but if we do, let's make Pr at some point for all commands (:

Reply:
@bwplotka Is there somewhere we already have the auto-generated one setup? Or would that be something new? If we want to proceed with that, is it worth creating an issue for it?